go/libkb/merkle_client.go

```diff
@@ -392,7 +656,25 @@ func (mc *MerkleClient) LastSeqno() Seqno {
 	mc.RLock()
 	defer mc.RUnlock()
 	if mc.lastRoot != nil {
-		return mc.lastRoot.seqno
+		return mc.lastRoot.Seqno()
+	}
+	return -1
+}
+
+func (mc *MerkleClient) LastRoot() *MerkleRoot {
+	mc.RLock()
+	defer mc.RUnlock()
+	if mc.lastRoot == nil {
+		return nil
+	}
+	return mc.lastRoot.ShallowCopy()
+}
+
+func (mc *MerkleClient) FirstSeqnoWithSkips() Seqno {
```

Comment:
Should we return a pointer here instead of relying on -1 to indicate "no value"? I'm worried about some code calling this function without realizing that it needs to check, doing a simple `>` comparison, and causing some confusing bug.